src/main/kotlin/no/nav/familie/tilbake/integration/kafka/KafkaProducer.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.familie.tilbake.integration.kafka
+
+import no.nav.familie.kontrakter.felles.historikkinnslag.OpprettHistorikkinnslagRequest
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.tilbake.config.Constants
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+class KafkaProducer(private val kafkaTemplate: KafkaTemplate<String, String>) {
+
+    private val log = LoggerFactory.getLogger(this::class.java)
+
+    fun sendHistorikkinnslag(behandlingId: UUID, key: String, request: OpprettHistorikkinnslagRequest) {
+        val melding = objectMapper.writeValueAsString(request)
+        val producerRecord = ProducerRecord(Constants.historikkTopic, key, melding)
+        kafkaTemplate.send(producerRecord)
```

Comment:
kan også bruke kafkaTemplate.send(Constants.historikkTopic, key, request).get(). Da vil det kastes en exception og tasken kan rekjøres om ønskelig

Reply:
Det kan gjøres slik også throw Feil(message = "Historikkinnslag kan ikke sendes for $behandlingId med $key. " +
                                                      "Feiler med ${it.message}",it)